chapters/object_types/instances.txt

```diff
@@ -46,24 +46,51 @@ and 16th float) as being (0, 0, 0, 1).
 | bounds   |`FLOAT32_BOX3`| axis-aligned bounding box in world-space (excluding the lights)
 |===================================================================================================
 
+[NOTE]
+.Note
+====
+Since the parameters of all other instance subtypes are supersets of the
+basic parameters listed above, it is recommend that the `transform` parameter
+should always be set in addition to any subtype specific transform parameters.
+====
+
 
 [[object_types_instance_transform]]
 ==== Transform
 
 Extension `KHR_INSTANCE_TRANSFORM`
 
 The `transform` instance subtype is a basic instance that implements the
-mandatory parameters and properties listed above. When this extension is
-supported creating an instance of unknown subtype will create an instance of
-subtype `transform` instead.
+parameters and properties listed above.
+
+[[extension_instance_transform_array]]
+The `KHR_INSTANCE_TRANSFORM_ARRAY` extension adds to the `transform` subtype
+additional types for the `transform` (and related) parameters by allowing
+arrays to do multi-instantiation. This gives applications a more efficient means
+to transform a single group many times by avoiding the need to create an entire
+object per-transform.
+
+.Additional <<api_concepts_parameters, parameters>> introduced by `KHR_INSTANCE_TRANSFORM_ARRAY` <<object_types_instance, instance>>.
+[cols="<4,<3,<11",options="header,unbreakable"]
+|==========================================================================================================
+| Name      | Type                       | Description
+| transform |`ARRAY1D` of `FLOAT32_MAT4` | array of matrices, instantiating `group` once per-element matrix
+| id        |`ARRAY1D` of `UINT32`       | <<frame_channels, extension `KHR_FRAME_CHANNEL_INSTANCE_ID`>>, optional user Id, for frame <<frame_channels, channel `instanceId`>>
+|==========================================================================================================
+
+The additional `id` array applies an instance identifier value to each element
+instance of the `transform` array. The sizes of these arrays must match. If the
+`id` array is not present when `transform` is set as an array, the single value
+version of `id` is used.
 
 [NOTE]
 .Note
 ====
-Since the parameters of all other instance subtypes are supersets of the
-`transform` subtype they can be safely substituted in case an implementation
-does not support that subtype. To maximize compatibility the transform parameter
-should always be set in addition to any subtype specific transform parameters.
+Since setting transforms using an array of matrices doesn't have a sensible
```

Comment:
Even in notes we should not use contractions: does not

Do we need the note at all (it is rather verbose)? It is clearly written already that it is a `KHR_INSTANCE_TRANSFORM_ARRAY` extension and what it adds, and of course it can be queried. But also fine.

Reply:
Addressed in latest revision.